peakinvestigator/lib/uploader.py

```diff
@@ -0,0 +1,94 @@
+## -*- coding: utf-8 -*-
+#
+# Copyright (c) 2016, Veritomyx, Inc.
+#
+# This file is part of the Python SDK for PeakInvestigator
+# (http://veritomyx.com) and is distributed under the terms
+# of the BSD 3-Clause license.
+from peakinvestigator.lib import *
```

Comment:
I like where you're going with the `peakinvestingator.lib` (i.e. classes not to be publicly used are in a subpackage). I  suggest `internal` instead of `lib`.